cli/src/ui/deployments.rs

```diff
@@ -144,4 +159,7 @@ pub fn add_deployment_to_kv_table(deployment: &Deployment, table: &mut Table) {
             &format!("{}: {}", header, value.to_str().unwrap_or("<BINARY>")),
         );
     }
+
+    table.add_kv_row("Min protocol version:", min_protocol_version);
+    table.add_kv_row("Max protocol version:", max_protocol_version);
```

Comment:
I would rather print those as the following:

* In case only one version supported: `table.add_kv_row("Protocol:", min_protocol_version);`
* In case range supported: `table.add_kv_row("Protocol:", format("[{} .. {}]", min_protocol_version, max_protocol_version));`

Reply:
Will change it.